src/resources/benefitCompany.ts

```diff
@@ -6,9 +6,9 @@ import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module
 export const BenefitCompanyResource: ResourceIF = {
   entityType: CorpTypeCd.BENEFIT_COMPANY,
   displayName: GetCorpFullDescription(CorpTypeCd.BENEFIT_COMPANY),
-  title: 'Benefit Company Statement:',
-  description: `This company is a benefit company and, as such, has purposes that include conducting its business
-        in a responsible and sustainable manner and promoting one or more public benefits.`,
```

Comment:
Using back-ticks for multi-line strings is neat, but while testing, the whitespace (including indents) must match exactly, forcing us to use weird indents in some places. So I changed this to concatenated individual strings in a few places.